tests/helpers/resolver.js

```diff
@@ -1,7 +1,7 @@
 import Resolver from 'ember/resolver';
 import config from '../../config/environment';
 
-var resolver = Resolver.create();
+let resolver = Resolver.create();
 
 resolver.namespace = {
   modulePrefix: config.modulePrefix,
```

Comment:
The rest of this file should following spacing and formatting standards. Change one change them all :-)